server/methods/core/groups.js

```diff
@@ -100,7 +100,7 @@ Meteor.methods({
 
     // prevent edits on owner. Owner groups is the default containing all roles, and as such should be untouched
     if (group.slug === "owner") {
-      throw new Meteor.Error(400, "Invalid request");
+      throw new Meteor.Error("bad-request", "Invalid request");
```

Comment:
Let's make this `invalid-parameter` and the long explanation a little more helpful if possible.